streaming/rrdpush.c

```diff
@@ -76,6 +77,7 @@ int rrdpush_init() {
     default_rrdpush_api_key     = appconfig_get(&stream_config, CONFIG_SECTION_STREAM, "api key", "");
     default_rrdpush_send_charts_matching      = appconfig_get(&stream_config, CONFIG_SECTION_STREAM, "send charts matching", "*");
     rrdhost_free_orphan_time    = config_get_number(CONFIG_SECTION_GLOBAL, "cleanup orphan hosts after seconds", rrdhost_free_orphan_time);
+    stream_stats_enabled        = (unsigned int)appconfig_get_boolean(&stream_config, CONFIG_SECTION_GLOBAL, "activate stream statistics", 0);
```

Comment:
Hi @stelfrag

The first time that I review your code I did not test it, because I saw we could miss some important information from the Operate System, after you fix that problem, today I began to test your PR.  In the first test that I executed, I decided to print the variable stream_stats_enabled inside the log files on Netdata master, and it was always set to zero independent of the position that I put  the parameter "activate stream statistics"  inside my stream.conf file, so I came here and I saw that you are mixing concepts. The "stream_config" variable is associated with "CONFIG_SECTION_STREAM" and here your are putting it with the CONFIG_SECTION_GLOBAL that is a section of the netdata.conf, please, fix this for I continue with the tests.

Reply:
Hi @thiagoftsm 

Thank you for your feedback. I am assuming you put a **[global]** section in the
*stream.conf*  (eg. /etc/netdata/stream.conf) file with a parameter 

```
[global]
    activate stream statistics = yes
```

The code block below is intended to look for the configuration parameter in the stream.conf under a *[global]* section

`stream_stats_enabled        = (unsigned int)appconfig_get_boolean(&stream_config, CONFIG_SECTION_GLOBAL, "activate stream statistics", 0);`

I will try to reproduce the behavior you are experiencing and analyze the logic of the
*appconfig_get_boolean* function to make sure it is behaving as I expected it to do
so I can provide solution

thank you !!

Reply:
Now I understood what you are trying to do better, so I will ask more two things to you. As you can see we have inside the directory streaming the file README.md, this is a file where we write our documentation, please, explain inside this file the new section in our stream.conf. No less important, inside the same directory(streaming) we also have a stream.conf file that is copied for our users when they use the command /etc/netdata/edit-config, add the newest section inside the file, I would prefer to insert on top, and comment it there.

Best Regards!

Reply:
Thanks for the feedback @thiagoftsm . I will update README.md and stream.conf